src/ils.cpp

```diff
@@ -101,6 +101,66 @@ usage() {
         "Usage: ils [-ArlLv] dataObj|collection ... ",
         "Usage: ils --bundle [-r] dataObj|collection ... ",
         "Display data Objects and collections stored in irods.",
+        " ",
+        "The following is typical output for ils with no arguments:"
+        "    $ ils",
+        "    /tempZone/public:",
+        "      big",
+        "      foo",
+        "The only information displayed are the names of data objects in the current",
+        "working directory.",
+        " ",
+        "The long format (given by -l) will display some more information:",
+        "    $ ils -l",
+        "    /tempZone/public:",
+        "      rods              0 demoResc     41943040 2021-04-29.17:57 & big",
+        "      rods              2 comp;arch    40236581 2021-04-19.03:00 X big",
```

Comment:
i think bump the filesize to the right a few spaces to align with the 'foos' below.

Reply:
Got it